docs/reference/contributing/guidelines/workflow.md

```diff
@@ -80,14 +102,28 @@ Each pull request goes through the following workflow:
 
 Labels that the Mbed OS maintainers add to a pull request represent the pull request workflow states. The Mbed OS maintainers are responsible for moving pull requests through the workflow states.
 
+Each state is time boxed. In most cases, this should be sufficient time to move to an another state.
+
 ##### Reviews
 
-All pull requests must be reviewed. The Mbed OS maintainers determine the most suitable person to review the pull request and tag that person accordingly. If a pull request requires a review from partners, the maintainers tag the corresponding GitHub group instead.
+All pull requests must be reviewed. The mbed CI bot determine the most suitable person to review the pull request and tag that person accordingly.
+
+Time: 3 days
 
 ##### The CI (Continuous Integration) testing
 
 There are a number of CI systems available. Which CI tests we run against a particular pull request depends on the effect of that pull request to the code base. Irrespective of which CIs tests run, Mbed OS has an all green policy, meaning that all the CI jobs that are triggered must pass before we merge the pull request.
 
+Time: 1 day
```

Comment:
Again, what do we mean by this? Are we saying this is the time limit between marking something as needs: ci and actually triggering it, or is it the time between triggering it and getting it to pass, or a combination of the two ?